tests/unit/core.js

```diff
@@ -872,6 +877,7 @@ exports.testConstModification = function (test) {
       .addError(8, "Attempting to override 'a' which is a constant.")
       .addError(8, "You might be leaking a variable (a) here.")
       .addError(53, "Missing '()' invoking a constructor.")
+      // .addError(17, "Attempting to override 'f' which is a constant.") TODO
```

Comment:
If a test that should pass is failing, you can add a `//TODO(name of person who should look at it): short description`, and if you want you can also comment out some tests that are failing. File a bug too, you can link to the bug in the comment.